src/response.rs

```diff
@@ -18,6 +18,12 @@ use serde::de::DeserializeOwned;
 #[cfg(feature = "charset")]
 use encoding_rs::Encoding;
 
+#[cfg(feature = "gzip")]
+use flate2::read::GzDecoder;
+
+#[cfg(feature = "brotli")]
+use brotli_decompressor::Decompressor;
```

Comment:
Minor: If we import this as `use brotli_decompressor::Decompressor as BrotliDecoder` so we get symmetry between `GzDecoder` and `BrotliDecoder`.

Reply:
Makes sense to me, done.